src/Http/Response.php

```diff
@@ -34,13 +34,20 @@
 
 /**
  * Responses contain the response text, status and headers of a HTTP response.
+ *
+ * There are external packages such as `http-message-util` that provide HTTP
```

Comment:
That is not the package name. Vendor part is missing.

Reply:
Thanks for catching that!